pkg/gotohelm/transpiler.go

```diff
@@ -160,18 +160,18 @@ func (t *Transpiler) transpileFile(f *ast.File) *File {
 		}
 
 		var params []Node
-		for _, param := range fn.Type.Params.List {
-			for _, name := range param.Names {
-				params = append(params, t.transpileExpr(name))
-			}
-		}
 		if fn.Recv != nil {
 			for _, param := range fn.Recv.List {
 				for _, name := range param.Names {
 					params = append(params, t.transpileExpr(name))
 				}
 			}
 		}
+		for _, param := range fn.Type.Params.List {
```

Comment:
super nit: Add a new line before the `for` loop. The code feels "cramped" otherwise.

You can ignore my comment about the ordering in the previous PR :D